app/src/keymap.c

```diff
@@ -19,8 +19,8 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 #include <zmk/events/layer_state_changed.h>
 #include <zmk/events/sensor_event.h>
 
-static zmk_keymap_layers_state_t _zmk_keymap_layer_state = 0;
-static uint8_t _zmk_keymap_layer_default = 0;
+static const uint8_t _zmk_keymap_layer_default = 0;
```

Comment:
I don't think we should be making this `const`, as there has been some ideas about changing the default layer on-the-fly (e.g., to change the keyboard from qwerty to colemak)

Reply:
The reason for const was to allow the static initialization in the following line.  I'd suggest leaving it for now and to change it in the future when the default layer can be non-0.  There are other places the code would need to be modified and this makes the current assumption explicit.  I say leave as-is.

Reply:
On the other hand if non-0 default has been test I should change it back.   My understanding is it is not being used or tested.

Reply:
Maybe we could `#DEFINE ZMK_KEYMAP_INITIAL_DEFAULT_LAYER 0` instead?

Reply:
okay